tests/foreman/api/test_repository.py

```diff
@@ -1425,6 +1425,42 @@ def test_positive_bulk_cancel_sync(self, target_sat, module_manifest_org):
             )
             assert result.status == 0
 
+    @pytest.mark.tier2
+    def test_positive_sync_repo_null_contents_changed(self, module_org, target_sat):
+        """test for null contents_changed parameter on actions::katello::repository::sync.
+
+        :id: f3923940-e097-4da3-aba7-b14dbcda857b
+
+        :expectedresults: After syncing a repo and running that null contents_changed
+            command, 0 rows should be returned(empty string)
+
+        :CaseImportance: High
+
+        :customerscenario: true
+
+        :BZ: 2089580
+
+        :CaseAutomation: Automated
+        """
+        with manifests.clone() as manifest:
+            upload_manifest(module_org.id, manifest.content)
```

Comment:
https://github.com/SatelliteQE/robottelo/blob/0018520356b29fe680bb931791bc55f17f0a76f0/pytest_fixtures/component/taxonomy.py#L62

Looks like this fixture would be appropriate for the test so you don't have to handle the manifest within the call phase.